src/expr/src/scalar/func.rs

```diff
@@ -3685,8 +3700,10 @@ pub enum UnaryFunc {
     MzRowSize,
 }
 
+derive_unary!(Not,);
```

Comment:
You want to add the names of all your structs into the array on this line:
https://github.com/MaterializeInc/materialize/blob/c5a2b20d195b16111e745eca3c0e8c9809c1a4df/src/expr-test-util/src/lib.rs#L38

To have the current state of the PR pass the tests, just change the aforementioned line from 
```
[AggregateExpr, ColumnOrder, ColumnType, RelationType]
```
to
```
[AggregateExpr, ColumnOrder, ColumnType, RelationType, Not]
```

Ping me if you have a macro for listing all the names of all the individual (mostly zero-arg) structs, and I'll see how I can have it work with `expr-test-util`.

Reply:
thank you! this worked